app/assets/images/payment_icons/wechatpay.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" role="img" viewBox="0 0 38 24" width="38" height="24" aria-labelledby="pi-wechatpay"><title id="pi-wechatpay">wechatpay</title><path d="M7.7 11.81a.72.72 0 0 1-.32.08.71.71 0 0 1-.56-.36v-.1l-2-4.19a.4.4 0 0 1 0-.15.35.35 0 0 1 .35-.35.38.38 0 0 1 .21.07l2.28 1.62a1 1 0 0 0 .57.17 1.15 1.15 0 0 0 .37-.06l10.62-4.8A11.3 11.3 0 0 0 10.58 0C4.74 0 0 4 0 8.82a8.28 8.28 0 0 0 3.66 6.66.71.71 0 0 1 .29.58.78.78 0 0 1 0 .22c-.18.67-.47 1.73-.48 1.78a1.22 1.22 0 0 0-.06.26.36.36 0 0 0 .36.35.47.47 0 0 0 .2-.06l2.29-1.37a1.08 1.08 0 0 1 .56-.13 1.06 1.06 0 0 1 .31 0 12.5 12.5 0 0 0 3.45.49c5.84 0 10.58-3.95 10.58-8.82a7.69 7.69 0 0 0-1.2-4.08L7.82 11.77z" fill="#27b34b"/></svg>
```

Comment:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" role="img" viewBox="0 0 38 24" width="38" height="24" aria-labelledby="pi-wechatpay"><title id="pi-wechatpay">WeChat Pay</title><path d="M7.7 11.81a.72.72 0 0 1-.32.08.71.71 0 0 1-.56-.36v-.1l-2-4.19a.4.4 0 0 1 0-.15.35.35 0 0 1 .35-.35.38.38 0 0 1 .21.07l2.28 1.62a1 1 0 0 0 .57.17 1.15 1.15 0 0 0 .37-.06l10.62-4.8A11.3 11.3 0 0 0 10.58 0C4.74 0 0 4 0 8.82a8.28 8.28 0 0 0 3.66 6.66.71.71 0 0 1 .29.58.78.78 0 0 1 0 .22c-.18.67-.47 1.73-.48 1.78a1.22 1.22 0 0 0-.06.26.36.36 0 0 0 .36.35.47.47 0 0 0 .2-.06l2.29-1.37a1.08 1.08 0 0 1 .56-.13 1.06 1.06 0 0 1 .31 0 12.5 12.5 0 0 0 3.45.49c5.84 0 10.58-3.95 10.58-8.82a7.69 7.69 0 0 0-1.2-4.08L7.82 11.77z" fill="#27b34b"/></svg>
```

Hi @nihaopay , can you make sure the label inside `<title id="pi-wechatpay">` follows the label in the `payment_icons.yml`? It should be `WeChat Pay`.